suite-common/message-system/config/config.v1.json

```diff
@@ -1,8 +1,84 @@
 {
     "version": 1,
-    "timestamp": "2023-11-13T00:00:00+00:00",
-    "sequence": 46,
+    "timestamp": "2023-12-06T00:00:00+00:00",
+    "sequence": 47,
     "actions": [
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": "!",
+                        "mobile": "<23.13.0",
+                        "web": "!"
+                    }
+                }
+            ],
+            "message": {
+                "id": "8231c0f4-e778-4838-bc50-5c29d9632f06",
+                "priority": 98,
+                "dismissible": true,
+                "variant": "critical",
+                "category": "feature",
+                "content": {
+                    "en-GB": "Update your app",
+                    "en": "Update your app",
+                    "es": "Update your app",
+                    "cs": "Update your app",
+                    "ru": "Update your app",
+                    "ja": "Update your app",
+                    "hu": "Update your app",
+                    "it": "Update your app"
+                },
+                "cta": {
+                    "action": "external-link",
+                    "link": "https://github.com/trezor/trezor-suite",
+                    "label": {
+                        "en-GB": "Update Suite",
+                        "en": "Update Suite",
+                        "es": "Actualizar Suite",
+                        "cs": "Aktualizovat Suite",
+                        "ru": "Обновить Suite",
+                        "ja": "Suiteのアップデート",
+                        "hu": "Frissítés Suite",
+                        "it": "Suite di aggiornamento"
+                    }
+                },
+                "feature": [
+                    {
+                        "domain": "killswitch",
+                        "flag": true
+                    }
+                ]
+            }
+        },
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": "!",
+                        "mobile": "<23.13.0",
+                        "web": "!"
+                    }
+                }
+            ],
+            "message": {
+                "id": "8231c0f4-e778-4838-bc40-5c28d9633f06",
+                "priority": 94,
+                "dismissible": true,
+                "variant": "critical",
+                "category": "banner",
+                "content": {
+                    "en-GB": "Good to see you",
```

Comment:
If we release this to develop, even temporarily, it's gonna show up for all the current users of the Suite Liteě, right? Is that the intention?

Reply:
There is not activated messages fetching in the previous build.

Reply:
🤔  But I might probably better make this more bulletproof in case we were wrong :)